lib/streamlit/report_session.py

```diff
@@ -171,6 +175,15 @@ def enqueue(self, msg):
             if scriptrunner is not None:
                 scriptrunner.maybe_handle_execution_control_request()
 
+        if msg.HasField("page_config_changed") and not self._set_page_config_allowed:
+            raise StreamlitAPIException(
+                "`streamlit.set_page_config` can only be called once, "
+                "as the first Streamlit command in your script."
```

Comment:
Small tweak:

```
`beta_set_page_config()` can only be called once per app, and only as the first Streamlit command in your script.

For more information refer to the [docs](https://docs.streamlit.io/en/stable/api.html#streamlit.beta_set_page_config).
```

which renders as:

> `beta_set_page_config()` can only be called once per app, and only as the first Streamlit command in your script.
> 
> For more information refer to the [docs](https://docs.streamlit.io/en/stable/api.html#streamlit.beta_set_page_config).
```

Reply:
Done! (Though I wonder about the difference between `script` and `app`...)